Automattic/wp-calypso
# Colour Schemes: Migrate Images to Webpack

#### Changes proposed in this Pull Request

Unfortunately, it's now necessary to migrate the colour scheme images to Webpack since they've been modified recently. This is currently what's bring displayed on WordPress.com. (they're very out of date) 

<img width="743" alt="Screenshot_2019-10-06_at_20 51 06-1" src="https://user-images.githubusercontent.com/43215253/66287403-98bd6700-e8cd-11e9-887e-35d736cfc61d.png">

Making this change isn't exactly ideal because the colour scheme image used to be automatically handled when adding the image, but having to add two extra lines shouldn't hurt and it does fix a bug. It's currently particularly bad as the Nightfall scheme received a new accent colour in #35949. The new scheme images should look like this. 

<img width="811" alt="Screenshot_2019-10-06_at_20 55 24-1" src="https://user-images.githubusercontent.com/43215253/66287448-c4d8e800-e8cd-11e9-94ce-4d651eb66de3.png">


#### Testing instructions



* Visit `/me/account` and verify the right image is appearing for the right scheme

cc @porada, @sixhours
